templates/circleci.yml.erb

```diff
@@ -0,0 +1,53 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/<%= app_name %>
+    docker:
+      - image: circleci/ruby:<%= Suspenders::RUBY_VERSION %>-node
+        environment:
+          RAILS_ENV: test
+          PGHOST: localhost
+          PGUSER: <%= app_name %>
+      - image: postgres:10.5
+        environment:
+           POSTGRES_USER: <%= app_name %>
+           POSTGRES_DB: <%= app_name %>_test
+           POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
```

Comment:
Give that you have have not spaced out the rest of this file, what are your thoughts on removing this whitespace?